dropbox/sdk.go

```diff
@@ -37,7 +37,7 @@ const (
 	hostAPI       = "api"
 	hostContent   = "content"
 	hostNotify    = "notify"
-	sdkVersion    = "5.4.0"
+	sdkVersion    = "5.0.0"
```

Comment:
Why this change? (Seems to be downgrading a few minor versions?)

Reply:
Great eye.  I didn't set a version environment variable before running the script. I have no idea how the numbers work (none of this is documented); I'll just choose 5.5